classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -170,9 +170,21 @@ public function manage()
     {
         $this->manageImages();
 
+        if (is_a($this->objOutput->getObjectRender(), WebserviceOutputJSONCore::class)) {
```

Comment:
```suggestion
        if ($this->objOutput->getObjectRender() instanceof WebserviceOutputJSONCore) {
```
Should be enough, calling functions is always slower than using language construct